src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Ball {
```

Comment:
The class `Ball` should have an access modifier. Consider using `public` for this class to ensure it is accessible from other packages if needed.